examples/ultralytics-yolov3/deepsparse-yolov3.ipynb

```diff
@@ -0,0 +1,308 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from deepsparse import compile_model\n",
+    "\n",
+    "sample_model_path = PRUNED_STUB  # replace with file path or stub of choice\n",
+    "model = compile_model(sample_model_path, batch_size=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Inference\n",
+    "The following cell runs pre-processing, inference, post-processing, and NMS on a sample image.  Try changing the `SAMPLE_IMAGE_IDX` (can be 0-19) to see different inputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy\n",
+    "from deepsparse_utils import YoloPostprocessor, postprocess_nms\n",
+    "\n",
+    "SAMPLE_IMAGE_IDX = 0\n",
+    "\n",
+    "sample_image = images[SAMPLE_IMAGE_IDX]\n",
+    "sample_image = sample_image.reshape(1, *sample_image.shape)\n",
+    "if sample_model_path != PRUNED_QUANT_STUB:  # preprocess images for non-quantized\n",
+    "    sample_image = sample_image.astype(numpy.float32) / 255.0\n",
+    "sample_batch = [numpy.ascontiguousarray(sample_image)]\n",
+    "\n",
+    "outputs = model(sample_batch)\n",
+    "outputs = YoloPostprocessor().pre_nms_postprocess(outputs)\n",
+    "outputs = postprocess_nms(outputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Annotate\n",
+    "The following cell visualizes the output from DeepSparse by annotating the original image with the returned bounding boxes and classes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "from deepsparse_utils import annotate_image\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "annotated_image = annotate_image(original_images[SAMPLE_IMAGE_IDX], outputs[0], {})\n",
+    "plt.imshow(annotated_image)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Benchmark\n",
+    "The following cells run benchmarks of the base, pruned, and pruned-quantized models.  Benchmark times are end-to-end and include pre-processing, inference, post-processing and NMS.  The defualt setting is a \"real-time\" batch-size 1 example with the maximum number of CPU cores available used. To try benchmarking other deployment scenarios, edit the `BATCH_SIZE`, `NUM_CORES`, and `IMAGE_SIZE` variables.\n",
+    "\n",
+    "The benchmarking function provided below calls into the `benchmark.py` script that is also provided in this examples directory.  For best results, that script should be used as processes in the jupyter notebook may interfere with the benchmark."
```

Comment:
```suggestion
    "The benchmarking function provided below calls into the `benchmark.py` script that is also provided in this examples directory.  For best results, that script should be used as processes in the Jupyter Notebook may interfere with the benchmark."
```

Reply:
@bfineran There's a word or two missing in that last sentence. e.g.,  should it read "as they may interfere"?